google-ads-examples/src/main/java/com/google/ads/googleads/examples/utils/CodeSampleHelper.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.utils;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+/** A helper class for all code examples. */
+public abstract class CodeSampleHelper {
+
+  /**
+   * Generates a printable string for the current date and time in UTC time zone.
+   *
+   * @return the result string.
+   */
+  public static String getPrintableDatetime() {
+    return new DateTime(DateTimeZone.UTC).toString("yyyyMMdd_HH:mm:ss.SSS") + "_UTC";
```

Comment:
WDYT about using the timezone of the machine so that the names are familiar to the user who ran the example? I'm thinking it could be confusing to look in the UI and see timestamps from another timezone.

Reply:
Agreed, changed.